spec/liquid_tags/github_tag/github_readme_tag_spec.rb

```diff
@@ -78,6 +80,13 @@ def generate_tag(path, options = "")
       end
     end
 
+    it "renders a repository with relative links in README" do
+      VCR.use_cassette("github_client_repository_relative_links") do
```

Comment:
I think the `github_client_repository_relative_links` file needs to be added to the repository as well :)

Reply:
@rhymes Done